lib/fhir/resources/binary.js

```diff
@@ -235,11 +236,7 @@ module.exports = (mongo) => {
           return callback(null, fhirCommon.buildHTTPOutcome(400, 'error', 'invalid', badRequest.message))
         }
 
-        const query = { $and: [] }
-
-        if (queryObject['contenttype']) {
-          query['$and'].push({ contentType: queryObject['contenttype'][constants.NO_MODIFER] })
-        }
+        const query = queryBuilder.buildQuery(ctx.resourceType, queryObject)
```

Comment:
Lets add in the customParams filter here for completeness, There are no custom params at the moment, but possible in future
`const filteredQueryObject = queryUtils.filterOutProperties(queryObject, Object.keys(customSupportedParams))`